lib/modules/versioning/distro.ts

```diff
@@ -49,6 +52,12 @@ export class DistroInfo {
         // istanbul ignore next
         continue;
       }
+
+      // istanbul ignore next: As of now, all entries in our data are released
+      if (!this.isReleased(obj.version)) {
+        continue;
+      }
```

Comment:
```suggestion
```

We can't do this here. When renovate runs longer than a day (hosted app, ws renovate on premise), then they never get never releases, so this check needs to be done on every call where apropriate

Reply:
`DistroInfo` constructor is no longer date dependent.
this check is now done only when needed inside `getNLatest`.